src/_help/specification-support/extensions.md

```diff
@@ -0,0 +1,20 @@
+---
+Title: Specification extensions
+---
+
+- TOC
+{:toc}
+
+These are additional properties that are not specified by either the OpenAPI or the AsyncAPI specifications but are helpful for you to be able to customize your documentation content. All those properties start with the x- naming convention to be identified as “eXternal” from the OpenAPI or AsyncAPI specification.
+
+### Add topics to your documentation (`x-topics`)
+
+We created a custom property to enrich your documentation. This vendor-specific property helpsto add more context paragraphs in your generated documentation. Find out more in our [dedicated section](/help/enhance-documentation-content/topics/).
+
+### Custom code sample examples (`x-codeSamples`)
+
+> This vendor extension is only available for an OpenAPI documents for now
+{: .info}
+
+This is another unsupported property by OpenAPI, we added a custom property so you can add your own code samples in one or more programming languages to your documentation. Find out more in our [dedicated section](/help/specification-support/doc-code-samples).
```

Comment:
```
This is another unsupported property by OpenAPI, we added a custom property so...
```

Not 100% sure of how this part starts. Like "another unsupported" feels a bit rough. WDYT?

Reply:
you are right.